ykcompile/src/lib.rs

```diff
@@ -1461,18 +1465,25 @@ impl<TT> TraceCompiler<TT> {
                     );
                 }
                 Location::Mem(..) => todo!(),
-                Location::Indirect { ptr, off } => match ptr {
```

Comment:
It's hard to see what this change was for. Can you elaborate?

Reply:
Sorry, it was difficult to pull those changes apart again so I left them in a single commit. This change is to get the `dumb_interpreter` test running. It implements the `todo` below.